go_backend/main.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+
+	"github.com/gin-gonic/gin"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+func homePage(c *gin.Context) {
+	c.JSON(200, gin.H{
+		"message": "Hello World",
+	})
+}
+
+// func postHomePage(c *gin.Context) {
+// 	body := c.Request.Body
+// 	value, err := ioutil.ReadAll(body)
+// 	if err != nil {
+// 		fmt.Println(err.Error())
+// 	}
+// 	c.JSON(200, gin.H{
+// 		"message": string(value),
+// 	})
+// }
+
+//sign up function
+func querySignUp(c *gin.Context) {
```

Comment:
Let's separate the login system from the main function since we will have a lot more functions need to add.